index.md

```diff
@@ -13,6 +13,11 @@ Paragraph of introductory material.
 >
 > A short paragraph describing
 > what learners need to do before tackling this lesson.
+> Importantly, you must link to the
+> setup instructions provided in `install.md`
+> and/or on the [workshop template][swc-workshop-template-rendered]
+> page.
```

Comment:
Maybe unpack this into more explicit detail?

> Importantly, you must link to installation instructions for any tools used in this lesson.
> Tools used by core lessons will be [documented in the workshop-template][1]; link there if your tool is listed.
> If your tool is not listed there, core-lesson maintainers should [file a workshop-template pull request adding it][2].
> Non-core-lesson maintainers should document the installation of non-core tools in a local `install.md`.

Or something like that.

[1]: https://swcarpentry.github.io/workshop-template/
[2]: https://github.com/swcarpentry/workshop-template/pulls